client/src/modules/cohort/components/Cohorts.jsx

```diff
@@ -22,7 +21,7 @@ import CollectionView from 'common/components/CollectionView';
 import FormDialog from 'common/components/FormDialog';
 import Breadcrumbs from 'common/components/Breadcrumbs';
 import { ColorType, Routes, ViewType } from 'common/constants/enums';
-import { enterView, leaveView } from 'sockets';
+import { enterView, leaveView } from 'common/model/actions';
```

Comment:
hmm, we have a general model directory in the app folder. Only wonder if we could we put this new action file there instead of creating a new model in the common directory?